UnitsNet/Scripts/UnitDefinitions/Flow.json

```diff
@@ -100,6 +100,18 @@
           "Abbreviations": [ "дм³/мин" ]
         }
       ]
+    },
+    {
+      "SingularName": "MillionStandardCubicFeetPerDay",
+      "PluralName": "MillionStandardCubicFeetPerDay",
+      "FromUnitToBaseFunc": "x*0.326992313",
```

Comment:
The reason we can't commit this is because "million standard cubic feet per day" implies a standard when, in fact, the standard is not defined.  Unfortunately, there are a number of standards, including differences between imperial and metric, that would need to be accommodated, making this a significantly more complicated conversion than a simple formula.